tests/golden/exoKafka/default.yaml

```diff
@@ -1,19 +1,20 @@
 apiVersion: exoscale.appcat.vshn.io/v1
-kind: ExoscaleKafka
+kind: exoscalekafka
```

Comment:
That looks wrong? Does it matter?

Reply:
Probably, I'll revert the whole lowercase thingy.